include/seqan3/io/alignment_file/output_format_concept.hpp

```diff
@@ -111,6 +114,7 @@ SEQAN3_CONCEPT AlignmentFileOutputFormat =
                   ref_id_type                            && ref_id,
                   ref_offset_type                        && ref_offset,
                   align_type                             && align,
```

Comment:
Why are they documented as `&&` but in the code they are `&`?

Reply:
well && on a template accepts both lvalues and rvalues, The concept only checks lvalues. 
Rvalues should probably be required too... But htis should be part of separate PR. I'll make a card

Reply:
Thanks!